README.md

```diff
@@ -73,6 +73,38 @@ be utilized by a light client implementation.
 │  └──  ...
  ```
 
+## BEEFY Key
+
+The current cryptographic scheme used by BEEFY is `ecdsa`. This is **different** from other keys being used in Substrate. Other
+Substrate related cryptographic keys are using the `sr25519` scheme. The most noticeable difference is that an `ecdsa` public key 
```

Comment:
```suggestion
The current cryptographic scheme used by BEEFY is `ecdsa`. This is **different** from other schemes like `sr25519` and `ed25519` which are commonly used in Substrate configuration for other pallets (BABE, GRANDPA, AuRa, etc). The most noticeable difference is that an `ecdsa` public key 
```

Reply:
While the previous is not technically incorrect, it's not like Substrate is bound to `sr25519`. It's just it was chosen over other crypto for default configuration, so it's commonly used, but the crypto is configurable.